lib/Frontend/CompilerInvocation.cpp

```diff
@@ -633,6 +643,27 @@ static bool ParseLangArgs(LangOptions &Opts, ArgList &Args,
                      A->getAsString(Args), A->getValue());
     }
   }
+  
+  // Check if the locale code is supported.
+  if (Arg *A = Args.getLastArg(OPT_locale)) {
+    std::string localeCode = A->getValue();
+    
+    auto result = llvm::find_if(supportedLocaleCodes, [&](const char * locale) {
```

Comment:
seems like we aren't really using the value in result there, so maybe `llvm::any_of` could be an option here :)

Reply:
Good point!